src/hooks/useActivityFeed/helpers.ts

```diff
@@ -19,7 +19,7 @@ const getActivityFeedMotionState = (
 
   const networkMotionState = motionStatesMap.get(action.motionData.motionId);
 
-  return networkMotionState
+  return networkMotionState !== undefined && networkMotionState !== null
```

Comment:
What's the reason for making this check stricter?

Reply:
Probably this wouldn't cause any issues but there is a motion state with a value `0` which would cause this condition to return undefined. If this was intended let me know
![image](https://github.com/JoinColony/colonyCDapp/assets/38806520/6a9d5ec9-ee7c-442a-aa4c-1902a7173d91)

Reply:
Yes it's fine for it to return undefined